src/main/kotlin/com/hibob/academy/resource/OwnerResource.kt

```diff
@@ -0,0 +1,86 @@
+package com.hibob.academy.resource
+
+import com.hibob.academy.types.Owner
+import com.hibob.academy.types.Pet
+import jakarta.ws.rs.*
+import jakarta.ws.rs.core.MediaType
+import jakarta.ws.rs.core.Response
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.RequestBody
+import java.util.*
+
+@Controller
+@Path("/api/noy/owners")
+@Produces(MediaType.APPLICATION_JSON)
+class OwnerResource {
+
+        // create v1
+        @POST
+        @Consumes(MediaType.APPLICATION_JSON)
+        @Path("/newOwner/v1")
+        fun addOwnerV1(@RequestBody owner: Owner): Response {
+            Response.status(Response.Status.CREATED).build()
+            return Response.ok(owner).build()
+        }
+
+        // create v2
+        @POST
+        @Consumes(MediaType.APPLICATION_JSON)
+        @Path("/newOwner/v2")
+        fun addOwnerV2(@RequestBody owner: Owner): Response {
```

Comment:
What's the difference between the 2? They both get the same Owner object. Maybe you can provide some extra implementation about how you use it differently?